package.json

```diff
@@ -18,12 +18,15 @@
     "@types/react-dom": "^17.0.11",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "js-yaml": "^4.1.0",
+    "lodash": "^4.17.21",
     "prettier": "^2.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "5.0.0",
-    "web-vitals": "^0.2.4"
+    "web-vitals": "^0.2.4",
+    "yaml": "^1.10.2"
```

Comment:
Is this dependency used? Of all the dependencies added I only see `js-yaml` used.

Reply:
Appears like `web-vitals` is required, but it seems odd that this wasn't an issue before. The docs seem to indicate its something [baked into CRA](https://create-react-app.dev/docs/measuring-performance/)